environment.py

```diff
@@ -16,6 +20,43 @@
 logger = logging.getLogger(__name__)
 
 
+def download_model_files(repo_url: str, repo_dir: str, local_path: str) -> None:
+    """Download or update model files (XML + meshes) from a GitHub repository.
+
+    :param repo_url: URL of the GitHub repository
+    :param repo_dir: Directory within the repository containing the model files
+    :param local_path: Local path where files should be saved
```

Comment:
[use google style comments](https://google.github.io/styleguide/pyguide.html)
```suggestion
    Args:
        repo_url: URL of the GitHub repository
        repo_dir: Directory within the repository containing the model files
        local_path: Local path where files should be saved
```